essentials.yml

```diff
@@ -1,6 +1,30 @@
 ---
+packaging:
+  config:
+    bundle:
+      groupId: "org.jenkins-ci.main"
+      artifactId: "jenkins-core-it"
+      vendor: "Jenkins project"
+      title: "Jenkins Integration Tests WAR"
+      description: "Jenkins WAR for integration tests"
+    war:
+      groupId: "org.jenkins-ci.main"
+      artifactId: "jenkins-war"
+      source:
+        dir: "."
 ath:
   useLocalSnapshots: false
   athRevision: "06caf67d37d945ec5228a80d73714fac692c28d3"
```

Comment:
Maybe YAGNI @raul-arabaolaza ?

Reply:
Absolutely, as updating this is going to become a bigger issue with time I have created [INFRA-1625|https://issues.jenkins-ci.org/browse/INFRA-1625] to start a discussion about the matter

Reply:
no, please use a deterministic version